include/ListView/ListView.php

```diff
@@ -1574,7 +1574,7 @@ function processListRows($data, $xtemplateSection, $html_varName)
         reset($data);
 
         //GETTING OFFSET
-        $offset = $this->getOffset($html_varName);
+        $offset = empty($this->getOffset($html_varName)) ? 0:$this->getOffset($html_varName);
```

Comment:
It would be cleaner to just typecast here.

Like 

`$offset = intval($this->getOffset($html_varName));`